app/models/repository_host/repository_host_data.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+# This class is meant to be a facade over the raw upstream data coming
+# from the different repository hosts' repository data. It's main goal
+# is to standardize the output from each repository host into a concrete
+# set of data so we can make sure the raw data is being mapped to the same
+# schema within the Libraries.io models and code.
+class RepositoryHost::RepositoryHostData
+  attr_reader :archived, :default_branch, :description, :fork, :full_name, :has_issues, :has_wiki, :homepage, :host_type,
+              :keywords, :language, :license, :name, :owner, :parent, :is_private, :repository_uuid, :scm, :size
+
+  def initialize(
+    archived:,
+    default_branch:,
+    description:,
+    fork:,
+    full_name:,
+    has_issues:,
+    has_wiki:,
+    homepage:,
+    host_type:,
+    keywords:,
+    language:,
+    license:,
+    name:,
+    owner:,
+    parent:,
+    is_private:,
+    repository_uuid:,
+    scm:,
+    size:,
+    **kwargs
+  )
+    @archived = archived
+    @default_branch = default_branch
+    @description = description
+    @fork = fork
+    @full_name = full_name
+    @has_issues = has_issues
+    @has_wiki = has_wiki
+    @homepage = homepage
+    @host_type = host_type
+    @keywords = keywords
+    @language = language
+    @license = license
+    @name = name
+    @owner = owner
+    @parent = parent
+    @private = is_private
```

Comment:
```suggestion
    @is_private = is_private
```

This error would not be possible using a Struct